src/components/MoviesBox/MoviesBox.js

```diff
@@ -5,14 +5,33 @@ import './MoviesBox.scss';
 export default class MovieBox extends Component {
   constructor() {
     super()
-    this.state = {}
+    this.state = {
+      cards: []
+    }
+  }
+
+  componentDidMount = () => {
+    fetch('https://swapi.co/api/films/')
+    .then(response => response.json())
+    .then(data => this.setState({cards: data.results.sort((a, b) => {return a.episode_id - b.episode_id})}))
```

Comment:
Maybe add one more .then( ) to sort response array and return it before setState is invoked?

Reply:
good call, just added it!